universal-application-tool-0.0.1/app/services/applicant/ApplicantData.java

```diff
@@ -302,7 +302,6 @@ private void putParentIfMissing(Path path) {
       return;
     }
 
-    // TODO(#624): get rid of this recursion.
```

Comment:
TODO is outdated as we chose to take a different design direction: https://github.com/seattle-uat/civiform/issues/624#issuecomment-845325307